wandb/sdk/data_types/html.py

```diff
@@ -19,16 +19,31 @@ class Html(BatchableMedia):
     """Wandb class for arbitrary html.
 
     Args:
-        data: (string or io object) HTML to display in wandb
+        data: (string or io object)
+            HTML can be initialized by providing either a path to a file,
+            or literal HTML in a string or io object.
```

Comment:
Since this is an argument description, it should start with a noun phrase describing the argument, so

> An IO object or string containing HTML, or a string that is a path to a file with the '.html' extension.